code/src/cljs/sixsq/nuvla/ui/utils/bulk_edit_tags_modal.cljs

```diff
@@ -0,0 +1,283 @@
+(ns sixsq.nuvla.ui.utils.bulk-edit-tags-modal
+  (:require [cljs.spec.alpha :as s]
+            [clojure.string :as str]
+            [re-frame.core :refer [dispatch reg-event-db reg-event-fx reg-sub
+                                   subscribe]]
+            [reagent.core :as r]
+            [sixsq.nuvla.ui.cimi-api.effects :as cimi-api-fx]
+            [sixsq.nuvla.ui.edges.events :as events]
+            [sixsq.nuvla.ui.i18n.subs :as i18n-subs]
+            [sixsq.nuvla.ui.main.components :as components]
+            [sixsq.nuvla.ui.plugins.table :as table-plugin]
+            [sixsq.nuvla.ui.utils.semantic-ui :as ui]
+            [sixsq.nuvla.ui.utils.semantic-ui-extensions :as uix]
+            [sixsq.nuvla.ui.messages.events :as messages-events]))
+
+
+(def modal-tags-set-id ::tags-set)
+(def modal-tags-add-id ::tags-add)
+(def modal-tags-remove-id ::tags-remove)
+(def modal-tags-remove-all ::tags-remove-all)
+(def tags-modal-ids [modal-tags-add-id modal-tags-set-id modal-tags-remove-id modal-tags-remove-all])
```

Comment:
you use `set(tags-modal-ids)` quite often, why not defining a 
`(def tags-modal-ids-set (set tags-modal-ids))`